sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -18,10 +19,13 @@ internal object RadarSettings {
     private const val KEY_METADATA = "user_metadata"
     private const val KEY_AD_ID_ENABLED = "ad_id_enabled"
     private const val KEY_TRACKING = "background_tracking"
+    private const val KEY_SERVER_TRACKING = "server_tracking_options"
```

Comment:
I think we need to remove `KEY_SERVER_TRACKING` in favor of `KEY_REMOTE_TRACKING_OPTIONS`

Reply:
Oh I see, this is to support "should listen".  On iOS the param name is a little more explicit `listenToServerTrackingOptions`, maybe we can do the same thing here.

Reply:
Used the string mentioned above, for clarity.